pycozmo/audio.py

```diff
@@ -78,13 +78,13 @@ def is_running(self):
         return self.thread.is_alive()
 
 
-def load_wav(self, filename: str):
+def load_wav(filename: str):
```

Comment:
`self` was causing exceptions

Reply:
Again, sorry about these mistakes!! and thank you!